pglast/printers/dml.py

```diff
@@ -656,9 +656,18 @@ def res_target(node, output):
 
 @node_printer('RowExpr')
 def row_expr(node, output):
-    output.write('ROW(')
-    output.print_list(node.args)
-    output.write(')')
+    coerciontype = enums.CoercionForm(node.row_format.value)
```

Comment:
Is there a particular reason why you did use this idiom (here and elsewhere, see `git grep 'enums\.[a-zA-Z]*('`) instead of a simpler
```
rf = node.row_format
if rf == enums.CoercionForm.COERCE_EXPLICIT_CALL:
  ...
elif rf == enums.CoercionForm.COERCE_IMPLICIT_CAST:
  ...
```

Reply:
No particular reason, just fixed that.
However, accessing the value member was necessary, at least for your CI to pass.

Reply:
Ok, thank you. I will try to understand the reason and will take care of that.
If you are done, I plan to merge your work into master in the following days.

Reply:
Probably something related to how enums are compared which might have evolved between versions of python ? The error I got was on python3.4 IIRC.
Unless you see other things to change, and apart from the manually-added enums which I guess should be automatically generated like the others, I think I'm done but tell me if I need to add test cases, comments or anything !